c7n/resources/connect.py

```diff
@@ -0,0 +1,59 @@
+from c7n.manager import resources
+from c7n.query import QueryResourceManager, TypeInfo
+from c7n.filters import ValueFilter
+from c7n.utils import local_session, type_schema
+
+
+@resources.register('connect')
+class Connect(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        service = 'connect'
+        enum_spec = ('list_instances', 'InstanceSummaryList', None)
+        arn_type = 'connect'
+        name = id = 'InstanceId'
+        universal_taggable = object()
+        cfn_type = 'AWS::Connect'
```

Comment:
afaics connect instances aren't supported by cloudformation per https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/AWS_Connect.html

in which case we can leave this blank.